### PR TITLE
PR #2 - CSS Edits, JavaScript Edits, PR#1 Fixes

### PR DESCRIPTION
**Pull request by Jamieatucker**
Next PR will include Server-Side changes.

### CODE REVIEW

File: portfolio/src/main/webapp/imagegallery.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      </figure>
+      <figure>
+        <img src="/images/jtcompsci.jpg" alt="Picture of me with computers in the background." class="center">
+        <figcaption>A Persona of Me Encapuslated in this Picture.</figcaption>
+      </figure>
+      <footer>
+        <a href="index.html" id="pvpage"> Previous</a>No copyright infringement is intended. <a href="hiddentalent.html" id="nxpage"> Next </a>
+      </footer>
+    </div>
+    </body>
```
Comment (joshmcadams) [inline portfolio/src/main/webapp/imagegallery.html:48]: I think this is two spaces too indented.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -13,16 +13,62 @@
...
+  factContainer.innerText = fact;
+}
+
+/**
+ * Adds the data from DataServlet using async/await (the return values are used directly).
+ */
+async function getDataUsingAsyncAwait(){
+    const response = await fetch('/data');
+    const data = await response.text();
+    const dataContainer = document.getElementById('data-container');
```
Comment (joshmcadams) [inline portfolio/src/main/webapp/script.js:45]:
```suggestion
    const dataContainer = document.querySelector('#data-container');
```

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -13,16 +13,62 @@
...
+    ' \n\nI created my first music mashup in late September 2018. I spent weeks using' + 
+    ' the program Audacity making sure it sounded as fluid as possible, with' + 
+    ' little training going into it. It may not have sounded the best, but I was thrilled' + 
+    ' to have created a mashup that I wanted, and had sounded coherent. The audio' + 
+    ' file below contains one of my mashups,' + 
+    ' called "Highest in the Woods Instrumental". Feel free to listen if you would' + 
+    ' like!';
+    
+    // Add it to the page
+    const hiddenTalentContainer = document.getElementById('hiddentalent-container');
```
Comment (joshmcadams) [inline portfolio/src/main/webapp/script.js:67]:
```suggestion
    const hiddenTalentContainer = document.querySelector('#hiddentalent-container');
    document.querySelector('#hiddentalent-container').style.visibility = 'visible';
    document.querySelector('#rollsafe').style.visibility = 'visible';
```

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -13,16 +13,62 @@
...
+    document.getElementById('hiddentalent-container').style.visibility = 'visible';
+    document.getElementById('rollsafe').style.visibility = 'visible';
+    hiddenTalentContainer.innerText = hiddenTalent;
+    
+    // Add audio file to the page
+    document.getElementById('music').style.visibility = 'visible';
```
Comment (joshmcadams) [inline portfolio/src/main/webapp/script.js:73]:
```suggestion
    document.querySelector('#music').style.visibility = 'visible';
```

File: portfolio/src/main/webapp/imagegallery.html (hunk 1 above)
Comment (Jamieatucker) [reply portfolio/src/main/webapp/imagegallery.html:48]: I looked into it and the </footer> tag is two spaces. So I think it's good.

**Pull request merged**